src/styles/blocks/star.scss

```
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.stars__star {
```

use the power of nesting
```suggestion 

  $__star {
```